docs/research/prompts/batch-23b-marten-ef-projection-registration-regression.md

```
@@ -0,0 +1,116 @@
+```csharp
+opts.Projections.Add(new OnboardingProjection(), ProjectionLifecycle.Inline);
+opts.Projections.Add(new UserProfileFromOnboardingProjection(), ProjectionLifecycle.Inline);
+```
+
+(Exact call site is in `backend/src/RunCoach.Api/Infrastructure/MartenConfiguration.cs` — read it before responding.)
+
+## What's broken
+
+Every integration test in `backend/tests/RunCoach.Api.Tests/` that boots `WebApplicationFactory<Program>` is failing at SUT host startup with:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)